qiita_pet/test/test_download.py

```diff
@@ -561,5 +564,33 @@ def test_download_sample_information(self):
         self.assertEqual(response.reason, 'You need to specify an artifact id')
 
 
+class TestDownloadPrivateArtifactHandler(TestHandlerBase):
+    def setUp(self):
+        super(TestDownloadPrivateArtifactHandler, self).setUp()
+
+    def tearDown(self):
+        super(TestDownloadPrivateArtifactHandler, self).tearDown()
```

Comment:
Are these calls necessary? I would imagine that if there's no changes to be made to setup/teardown then the test runner can use the super class's methods.

Reply:
I don't know.  I'm fairly certain they don't hurt anything at least.  This is the pattern used by every class in the test_download file.

Reply:
If these are not necessary then I would remove them. Looks like only a few of sub-classes are doing additional work after calling super. The other classes probably copy-pasted the code? @antgonza do you know if this is necessary?

Reply:
No idea, what about removing from here and check if everything works as expected?

Reply:
That sounds like a good idea to me, make sure to count the same number of tests that are being executed, and that you can run the test suite a couple of times (to ensure there's no side effects).

Reply:
Removed them, 12 tests before, 12 tests after, no apparent side effects.  Looks like they're unnecessary.

Reply:
Wonderful, thanks for looking into this!